pennylane/drawer/mpldrawer.py

```diff
@@ -591,6 +592,14 @@ def ctrl(self, layer, wires, wires_target=None, control_values=None, options=Non
         min_wire = min(wires_all)
         max_wire = max(wires_all)
 
+        min_target, max_target = min(wires_target), max(wires_target)
+        if min_target < min(wires_ctrl) < max_target or min_target < max(wires_ctrl) < max_target:
```

Comment:
Would this fail in the case where `wires_ctrl = [0, 2, 4]` and `wires_target = [1, 3]`? The control wire `2` is hidden but the warning isn't raised.